dotCMS/src/main/java/com/dotmarketing/tag/business/TagFactoryImpl.java

```diff
@@ -446,6 +446,39 @@ public void deleteTag ( Tag tag ) throws DotDataException {
         dc.loadResult();
     }
 
+    @Override
+    public void deleteTagsInBatch(java.util.Collection<String> tagIds) throws DotDataException {
```

Comment:
please add a test that covers this code on TagAPITest
test public void deleteTags(String... tagIds) throws DotDataException on the API